### DISCUSSION TITLE
Do you think since it's in beta, it's an error or bugged?

### DISCUSSION THREAD

**Discussion by ainsleyrehn** [leightonavantgardehaus/.github#11] [Q&A] [answered]
Although, I’m not sure if it's not enough documentation or a lack of understanding on yaml on my part. The branches workaround seems fine for now but not properly descriptive IMO.

**Comment by TL8N** [comment, marked as answer]
Neither it's resolved. I just ran to deployments and both passed credential.  

I do think something is bugged though....
